vms/platformvm/state/state.go

```diff
@@ -215,10 +215,11 @@ type stateBlk struct {
 type state struct {
 	validatorUptimes
 
-	cfg     *config.Config
-	ctx     *snow.Context
-	metrics metrics.Metrics
-	rewards reward.Calculator
+	cfg              *config.Config
+	ctx              *snow.Context
+	metrics          metrics.Metrics
+	nodeStakeLoggers []validators.NodeStakeLogger
```

Comment:
I don't think we ever use this. Can we just delete it?

Reply:
Done.